icu4c/source/i18n/unicode/basictz.h

```diff
@@ -43,6 +43,9 @@ class U_I18N_API BasicTimeZone: public TimeZone {
      */
     virtual ~BasicTimeZone();
 
+    /** Default copy assignment (inherited) */
```

Comment:
What version number?

Reply:
these were default pre-existing. So i think it should be exactly the version and stability of the copy c'tor, even if it was stable.

Reply:
right, looks like stable ICU 3.8 here

i suspect that this one (and maybe some others) might have been exposed since my icu 65 changes for polymorphic clone